scripts/main.js

```diff
@@ -116,11 +121,19 @@ var listingImageDisplay = function(pin) {
     return listingImage;
 };
 
+var iconBase = 'https://maps.google.com/mapfiles/kml/shapes/';
+var icons = {
+    sunny: {
+      icon: iconBase + 'sunny.png'
+    },
+  };
+
 var addMarkersToMap = function (infoWindow) {
     pins.forEach( function (pin) {
         var marker = new google.maps.Marker( {
             position: pin.position,
             map: map,
+            icon: icons[pin.type].icon,
```

Comment:
This could probably be refactored so we only need 1 line of code to get it working:

icon: 'https://maps.google.com/mapfiles/kml/shapes/sunny.png'